libr/main/radiff2.c

```diff
@@ -1236,7 +1233,6 @@ R_API int r_main_radiff2(int argc, const char **argv) {
 		}
 		r_diff_free (d);
 		break;
-	case MODE_DIST:
```

Comment:
Since you are removing this, could you remove the
```
} else {
   d->type = 0;
}
```
few linew below? I don't think it can be ever executed.